cetmix_tower_server/tests/test_plan.py

```diff
@@ -322,16 +322,50 @@ def test_plan_user_access_rule(self):
         )
         self.assertTrue(self.plan_3)
```

Comment:
What does this assertion mean?

Reply:
Although this change was not made in my last commit,
I can assume that self.assertTrue(self.plan_3) will pass if self.plan_3 is not None or not an empty value. 
So this will confirm that self.plan_3 exists and that the record was created successfully.